src/parser_urdf.cc

```diff
@@ -2655,42 +2657,58 @@ void URDF2SDF::ListSDFExtensions(const std::string &_reference)
   }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+void _DisplayDbgOrWarning(const std::string &_msg,
+                          const bool _enableNewWarnings) {
+  if (_enableNewWarnings)
+    sdfwarn << _msg;
+  else
+    sdfdbg << _msg;
```

Comment:
this looks like an enforcement policy to me, so I think we could use that mechanism explicitly, but only the `LOG` and `WARN` options

Reply:
Changed in [b05e7b7](https://github.com/gazebosim/sdformat/pull/1192/commits/b05e7b72b2b48277c026b7290166eaffc0cd074a)